packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -11,10 +11,10 @@ import 'package:gotrue/src/types/auth_response.dart';
 import 'package:gotrue/src/types/fetch_options.dart';
 import 'package:http/http.dart';
 import 'package:jwt_decode/jwt_decode.dart';
+import 'package:logging/logging.dart';
 import 'package:meta/meta.dart';
 import 'package:retry/retry.dart';
 import 'package:rxdart/subjects.dart';
-import 'package:logging/logging.dart';
 
 import 'broadcast_stub.dart' if (dart.library.html) './broadcast_web.dart'
```

Comment:
This has to be changed as well, doesn't it?

Reply:
I think this import just got moved to line 14 automatically from my IDE.

Reply:
No, I meant the `dart.library.html` import: https://dart.dev/interop/js-interop/package-web#conditional-imports

Reply:
Should be good now 👍